type-tests/api.ts

```diff
@@ -45,6 +46,11 @@ import {
   unsetContext,
   teardownContext,
   setupRenderingContext,
+  BaseContext,
+  TestContext,
+  RenderingTestContext,
+  TestMetadata,
+  DebugInfo as InternalDebugInfo,
```

Comment:
No harm done here, but for future reference: I try to keep things like these ordering changes separate from any *meaningful* changes—I just pull them into a separate commit, using `git add --patch` (though I *always* do that via a GUI because I find the text interface for it terrible)—so that it's clear what the meaningful change was. Here, for example, it took some extra reading and comparison to figure out that `RenderingTestContext` was the key change in this set.

Reply:
I'll just un-sort for now.

Re `RenderingTestContext`, should it be made public to consumers of this library? Currently it's not actually exported, so this import doesn't even work. I fix that in https://github.com/emberjs/ember-test-helpers/pull/1286, where the same issue came up for different reasons.

Reply:
Per my note on #1286, I think exporting it is great. 👍🏼